x-pack/plugins/apm/public/components/app/service_inventory/service_list/order_service_items.test.ts

```diff
@@ -191,40 +191,130 @@ describe('orderServiceItems', () => {
   });
 
   describe('when sorting by alphabetical fields', () => {
-    const sortedItems = orderServiceItems({
-      primarySortField: ServiceInventoryFieldName.ServiceName,
-      sortDirection: 'asc',
-      tiebreakerField: ServiceInventoryFieldName.ServiceName,
-      items: [
-        {
-          serviceName: 'd-service',
-          healthStatus: ServiceHealthStatus.unknown,
-        },
-        {
-          serviceName: 'a-service',
-          healthStatus: ServiceHealthStatus.unknown,
-        },
-        {
-          serviceName: 'b-service',
-          healthStatus: ServiceHealthStatus.unknown,
-        },
-        {
-          serviceName: 'c-service',
-          healthStatus: ServiceHealthStatus.unknown,
-        },
-        {
-          serviceName: '0-service',
-          healthStatus: ServiceHealthStatus.unknown,
-        },
-      ],
+    it('sorts correctly', () => {
```

Comment:
This existing test block was missing the `it` block, due to which it was not executing